app/facades/member_search_facade.rb

```diff
@@ -0,0 +1,18 @@
+class MemberSearchFacade
+  def initialize(params)
+    @params = params
+    @state = params[:state]
+  end
+
+  def get_member_details
+    bill_sponsors = BillSearchFacade.new(@params).bill_sponsors
+    sponsor_details = MemberService.new(@params).rep_details(@state, bill_sponsors)
+    sponsor_response = sponsor_details[:sponsor_details][:results]
+    sponsor_response.map do |sponsor|
+      sponsor_hash = { 
+        
+      }
+    end
+    member = Member.new(sponsor_hash)
+  end
```

Comment:
we may want to comment out this method or delete it since it is being worked on on another branch, to avoid merge conflicts in the future